kinto/api-test/mockclient.py

```diff
@@ -4,7 +4,6 @@
 
 stage_url = "https://kinto.stage.mozaws.net/v1/"
 
-
```

Comment:
it's a silly pep8 standard, but nevertheless it is https://www.python.org/dev/peps/pep-0008/#blank-lines